src/bin/gtktest.rs

```diff
@@ -112,8 +112,12 @@ mod example {
             let window = upgrade_weak!(window_weak);
             let entry = upgrade_weak!(entry_weak);
 
-            let dialog = Dialog::new_with_buttons(Some("Hello!"), Some(&window), gtk::DialogFlags::MODAL,
-                &[("No", 0), ("Yes", 1), ("Yes!", 2)]);
+            let dialog = Dialog::new_with_buttons(Some("Hello!"),
+                                                  Some(&window),
+                                                  gtk::DialogFlags::MODAL,
+                                                  &[("No", ResponseType::Other(0)),
+                                                    ("Yes", ResponseType::Other(1)),
+                                                    ("Yes!", ResponseType::Other(2))]);
```

Comment:
Maybe better declare global statics or constants?

Reply:
Didn't think about it but I could yes. It just seems a bit too much something used only once. What do you think @sdroege?

Reply:
There are `ResponseType::Yes` and `::No`, why not use those? :) And for the third one, maybe just call it `"Custom"` and use `::Other(0)` or so for that?